tests/transforms/test_tape_expand.py

```diff
@@ -624,12 +623,12 @@ def circuit():
             return qml.expval(qml.PauliZ(0))
 
         # BasicEntanglerLayers custom decomposition involves AngleEmbedding. If
-        # expansion depth is 1, the AngleEmbedding will still be decomposed into
+        # expansion depth is 2, the AngleEmbedding will still be decomposed into
         # RX (since it's not a supported operation on the device), but the RX will
         # not be further decomposed even though the custom decomposition is specified.
```

Comment:
This was not being properly tested before, and I think the logic in this comment is incorrect, it should be `depth=2`.